data/json/npcs/missiondef.json

```
@@ -577,6 +577,17 @@
     "start": "find_safety",
     "origins": [ "ORIGIN_NULL" ]
   },
+  {
+    "id": "MISSION_LAST_DELIVERY",
+    "type": "mission_definition",
+    "name": { "str": "Reach The Mansion And Finnish This Delivery" },
```

```suggestion
    "name": { "str": "Reach The Mansion And Finish This Delivery" },
```
_Insert joke about Finland not existing here_